biomage/experiment/download.py

```diff
@@ -4,17 +4,13 @@
 
 import boto3
 import click
+from botocore.exceptions import ClientError
 
 from ..rds.run import run_rds_command
-from ..utils.constants import (
-    CELLSETS_BUCKET,
-    DEFAULT_AWS_PROFILE,
-    FILTERED_CELLS_BUCKET,
-    PROCESSED_FILES_BUCKET,
-    RAW_FILES_BUCKET,
-    SAMPLES_BUCKET,
-    STAGING,
-)
+from ..utils.constants import (CELLSETS_BUCKET, DEFAULT_AWS_PROFILE,
```

Comment:
this is not the correct formatter that we use. Could you install the correct one? Otherwise you can use the command in the makefile

Reply:
formatter changed